package.json

```diff
@@ -36,6 +36,7 @@
   },
   "dependencies": {
     "redux": "4.0.1",
+    "react-redux": "7.0.3",
     "@fortawesome/fontawesome-svg-core": "1.2.18",
```

Comment:
<img width="908" alt="Screen Shot 2019-05-15 at 6 23 17 PM" src="https://user-images.githubusercontent.com/4022631/57819744-9734ac00-773e-11e9-89fc-edda8ac47112.png">

should get rid of this dep, or configure the ssr for it